docs/configuration/client.md

```diff
@@ -88,7 +88,7 @@ license: |
 | celeborn.client.shuffle.batchHandleCommitPartition.threads | 8 | false | Threads number for LifecycleManager to handle commit partition request in batch. | 0.3.0 | celeborn.shuffle.batchHandleCommitPartition.threads | 
 | celeborn.client.shuffle.batchHandleReleasePartition.interval | 5s | false | Interval for LifecycleManager to schedule handling release partition requests in batch. | 0.3.0 |  | 
 | celeborn.client.shuffle.batchHandleReleasePartition.threads | 8 | false | Threads number for LifecycleManager to handle release partition request in batch. | 0.3.0 |  | 
-| celeborn.client.shuffle.compression.codec | LZ4 | false | The codec used to compress shuffle data. By default, Celeborn provides three codecs: `lz4`, `zstd`, `none`. | 0.3.0 | celeborn.shuffle.compression.codec,remote-shuffle.job.compression.codec | 
+| celeborn.client.shuffle.compression.codec | LZ4 | false | The codec used to compress shuffle data. By default, Celeborn provides three codecs: `lz4`, `zstd`, `none`. `none` means that shuffle compression is disabled. Since Flink version 1.17, zstd is supported for Flink shuffle client. | 0.3.0 | celeborn.shuffle.compression.codec,remote-shuffle.job.compression.codec | 
```

Comment:
Here I have a minor concern regarding the `lzo` compression codec. Since Flink 1.17, `lzo` codec has also been supported, should we export that compression codec as well? 
@RexXiong could you please take a look at this?

Reply:
@TanYuxin-tyx, the shuffle client of Celeborn does not support LZO codec, therefore this could not export this compression codec.

Reply:
> @TanYuxin-tyx, the shuffle client of Celeborn does not support LZO codec, therefore this could not export this compression codec.

Agree, If the community needs support lZO or other compression codec, better open another ticket to resolve.

Reply:
OK, make sense.

Reply:
@TanYuxin-tyx I think we should be relatively conservative here. The shuffle data is a kind of application internal transient data format and the lifecycle is limited to one application, if there is no benefit(e.g. faster, or higher compression rate) from supporting a new codec, we should not do that.

Reply:
Additionally, vanilla Spark supports lz4, lzf, snappy, zstd codec for shuffle data compression, but only lz4 and zstd are adopted in Celeborn, because

1. lz4 is fastest
2. zstd has the best overall compression ratio and compression speed, a little bit slower than lz4, but ~2x compression ratio of lz4

Reply:
@pan3793 Thanks for joining the discussion. Considering the compression efficiency and the alignment with Spark shuffle, it sounds ok to not export `lzo` codec.